packages/salesforcedx-vscode-lightning/package.json

```diff
@@ -28,7 +28,7 @@
     "aura-language-server": "2.1.0",
     "change-case": "^3.1.0",
     "lightning-lsp-common": "2.1.0",
-    "open": "^6.0.0",
+    "open": "6.0.0",
```

Comment:
@AnanyaJha this fixes the compile issue we're seeing this morning with using version 6.2.0.